### ISSUE TITLE
External links are not indicated on screen readers

### ISSUE THREAD

**Issue by amyleadem** [uswds/uswds#5164]
## Description

External links have no screen reader identifier to match the “external link” icon. 

This issue was created in response to this [related Slack thread](https://gsa-tts.slack.com/archives/C3F14AHSQ/p1649690481675279).

## Steps to reproduce the issue

1. Open any page on https://designsystem.digital.gov/
2. Activate screen reader
3. Navigate to any external link (indicated by the external link icon)
![image](https://user-images.githubusercontent.com/93996430/162980945-ef83d6fe-8b47-4f5b-943c-6e5b18ca4d5f.png)
5. Check if the readout gives indication that the link is pointing to an external url

### Voiceover:
![image](https://user-images.githubusercontent.com/93996430/162981590-bf2914b2-bf81-4f7a-b479-f3691a7a5985.png)

### NVDA:

NVDA also failed to mention it was an external link

Screen Reader: NVDA
System: Windows 10 Pro
Browser: Chrome

**Comment by mejiaj** [comment]
Moving back to `In Progress` because of new requirements:

- [ ] External links in new windows must be announced
- [ ] External links in same tab must be announced
